### DISCUSSION TITLE
Data from Past Seasons

### DISCUSSION THREAD

**Discussion by IPetTheCats** [tbrittain/SmbExplorerCompanion#129] [Ideas]
Hey Trey!

Awesome work. Can't wait to see where you take this project.

I know you made mention in the FAQ about future development regarding the exporting/importing of current season data for past seasons. If the challenges you've encountered thus far don't completely rule that out, I'd be delighted if you were able to accommodate that feature.

I've got one season under my belt with an existing franchise (that I don't have exported data for) that I'd love to bring into the companion app.

Thanks again for you hard work!

**Comment by tbrittain** [comment]
Hi there! Speaking from memory, but I think we should be able to export reasonably accurate batting/pitching stats going back pretty far (probably the 50 season limit as of SMB4), but I am less confident about team regular season (and maybe postseason) schedules. I need to take a look at all of this again and reassess, but I agree it would be a great thing to support
